### ISSUE TITLE
Atom latex packages - which one to use?

### ISSUE THREAD

**Issue by Kriechi** [James-Yu/Atom-LaTeX#166]
Hi,

I'm recently having problems with the [latextools](https://github.com/msiniscalchi/atom-latextools) package, which seems to be abandoned.
Now I'm looking to modernize my latex integration in Atom with a new package (or multiple ones).

I found the package [latex](https://github.com/thomasjo/atom-latex), which seems to provide similar functionally. I would like to see the Atom & LaTeX community to undergo some kind of unifying process, as already mentioned in some of the READMEs on various projects.

As the readme already mentions, James-Yu has moved on, leaving @ashthespy in charge.

@ashthespy and @thomasjo, would you mind briefly describing any differences between your two projects, and why you think there is a need for your project in specific? (leaving out the 5+ smaller ones out there...)

This might sound like a sales pitch, but I honestly would like to get the ball rolling. Are there any features that are worth porting over to the other project?

Thanks!

**Comment by thomasjo** [comment]
I don't know anything about this project. I created `latex` back in the early days of Atom since there was nothing else back then. I created it to scratch my own itch, but decided that since other's might find it useful that it was worth sharing it as a community package. 

Why this package (`atom-latex`) was created instead of contributing to `latex` is only something @James-Yu can answer I suppose. While I'm a bit too busy with my research work (i.e. real job) these days to do much besides fixing critical bugs and triage issues, I have not intention of abandoning my package since I still use it every day for writing papers, etc.

**Comment by ashthespy** [comment]
I was having issues with high dpi screens and TeXworks - so I decided to give Atom a spin. To be honest - I initially used @thomasjo 's `latex` package - but didn't like a few things at the time (external pdf viewer, custom toolchains was limited to `latexmk`) so landed up here, and nipped and tucked at the package to suit my workflow. 
`atom-latex` is pretty much my daily drive - so bug fixes should find there way here - but feature requests are subject to my free time.

But while brainstorming here - @James-Yu do you think we could modularise  both `atom-latex` and `latex-workshop` into latex specific (autocomplete, log parsing, etc) and editor specific api?

**Comment by kylebarron** [comment]
I agree about merging the syntax highlighting in this package and `language-latex`. They've diverged a bit and it's easier to keep track of only one package.

I would recommend merging the syntax highlighting changes from this package into `language-latex`, then removing the syntax highlighting files from this package, and requiring `language-latex` as a dependency when a user installs this package. You can use [atom-package-deps](https://www.npmjs.com/package/atom-package-deps) to do this.

**Comment by JackSwett** [comment]
@kylebarron Stop making sense. No room for that on the internet.

**Comment by TiriaAndersen** [comment]
This sounds cool, would love to know how things are going.
